src/deluge/io/midi/midi_takeover.h

```diff
@@ -25,8 +25,16 @@ class MidiTakeover final {
 public:
 	MidiTakeover();
 
-	int32_t calculateKnobPos(ModelStackWithAutoParam* modelStackWithParam, int32_t knobPos, int32_t value,
-	                         MIDIKnob* knob = nullptr, bool doingMidiFollow = false, int32_t ccNumber = MIDI_CC_NONE);
+	int32_t calculateKnobPos(int32_t knobPos, int32_t value, MIDIKnob* knob = nullptr, bool doingMidiFollow = false,
```

Comment:
There's nothing in this class, it should just be a namespace or normal functions instead of being attached to a class with no member data

The private function declarations would go in the top of the cpp file instead of the header file